### ISSUE TITLE
Directional arrow control to the left/right edge of screen

### ISSUE THREAD

**Issue by Danniemite** [fancyapps/fancybox#1287]
Hello! 

I think it would be nice, if you could provide a - maybe optional - screen edge arrow control to the Fancybox.  Intuitively I look for the screen edges automatically, right after the browser opens. 
What do you think?

Thank you for this great code!
Daniel

**Comment by fancyapps** [comment]
Hi, Thanks!

Yes, I`ll add them at some point. I actually waited for the first person to ask about that :)

Well, you said it yourself - you automatically start looking at the seen edges. Don`t you find it annoying that every time you start a lightbox you have to look where to click next? I do and therefore I did not create those links, it is so much easier to just swipe or click buttons at the top toolbar (those arrows always stay at the same place and you no not need to move mouse so much when using big screen).

**Comment by Danniemie** [comment]
I see your point, and respect it. Sounds reforming, but my point is from the 'click' user experience. Clicking (& searching for) small buttons always gives me the feeling like if I would be trapped in a small place. :) But returning to your opinion: that's why I recommended this maybe as an optional feature, that you could turn on from JS. ;)

**Comment by fancyapps** [comment]
Yes, of course, therefore I`ll add them optionally. 

But, so much to do... so many ideas ..  and so many support requests, therefore I can`t tell when exactly this will be implemented.

**Comment by Cu8eeeR** [comment]
+1, this is the only feature which I miss in FancyBox :)

**Comment by mjcampagna** [comment]
I never asked, but it was one of the first customizations I made to my own implementation. This was easy to do via the 'baseTpl' option, and some custom CSS. I then use [the script I posted previously](https://github.com/fancyapps/fancybox/issues/1214#issuecomment-277585989) to show/hide the arrows based on mouse movement.

**Comment by Meedum** [comment]
Hello!

After I installed this script on my site the following indicators have deteriorated:
Average Page depth: before~10p. after~7p.  
Average Time on site: before~5m after~4m30s
I myself very much like it, but visitors to the site do not like something. Probably looking for arrows:)

Look at how images are viewed in search engines
Google - only arrows
Bing - only gestures
Yahoo - arrows and gestures! 

I like how it is done in Yahoo, only the arrows must be made invisible. 
I think that it will be good to make invisible arrows instead of zoom.  Especially the zoom does not work correctly in older browsers.

Thank you for this nice script!
Your script is the best of all lightboxes and I hope it gets even better.

Sorry for the bad English, I hope you understand what I wrote :)
Eduard
